.github/workflows/r_package.yml

```diff
@@ -258,58 +263,78 @@ jobs:
+          - clang17
+          - clang18
+          - clang19
+          - gcc14
+          - intel
     runs-on: ubuntu-latest
-    container: rhub/debian-clang-devel
-    env:
-      DEBIAN_FRONTEND: noninteractive
+    container: ghcr.io/r-hub/containers/${{ matrix.image }}:latest
     steps:
-      - name: Install Git before checkout
-        shell: bash
-        run: |
-          apt-get update --allow-releaseinfo-change
-          apt-get install --no-install-recommends -y git
-      - name: Trust git cloning LightGBM
-        run: |
-          git config --global --add safe.directory "${GITHUB_WORKSPACE}"
       - name: Checkout repository
         uses: actions/checkout@v4
         with:
           fetch-depth: 5
           submodules: true
-      - name: install clang
+      - name: Install pandoc
+        uses: r-lib/actions/setup-pandoc@v2
+      - name: Install LaTeX
+        shell: bash
         run: |
-          ./.ci/install-clang-devel.sh ${{ matrix.clang-version }}
+          if type -f apt 2>&1 > /dev/null; then
+            apt-get update
+            apt-get install --no-install-recommends -y \
+                devscripts \
+                texinfo \
+                texlive-latex-extra \
+                texlive-latex-recommended \
+                texlive-fonts-recommended \
+                texlive-fonts-extra \
+                tidy \
+                qpdf
+          else
+            yum update -y
+            yum install -y \
+                devscripts \
+                qpdf \
+                texinfo \
+                texinfo-tex \
+                texlive-latex \
+                tidy
+          fi
       - name: Install packages and run tests
         shell: bash
         run: |
-          export PATH=/opt/R-devel/bin/:${PATH}
           Rscript -e "install.packages(c('R6', 'data.table', 'jsonlite', 'knitr', 'markdown', 'Matrix', 'RhpcBLASctl', 'testthat'), repos = 'https://cran.rstudio.com', Ncpus = parallel::detectCores())"
           sh build-cran-package.sh
-          R CMD check --as-cran --run-donttest lightgbm_*.tar.gz || exit 1
-          echo ""
-          echo "install logs:"
-          echo ""
-          cat lightgbm.Rcheck/00install.out
-          echo ""
-          if grep -q -E "NOTE|WARNING|ERROR" lightgbm.Rcheck/00check.log; then
-              echo "NOTEs, WARNINGs, or ERRORs have been found by R CMD check"
-              exit 1
-          fi
+          # allowing the following NOTEs (produced by default in the clang images):
+          #
+          #   * checking compilation flags used ... NOTE
+          #       Compilation used the following non-portable flag(s):
+          #       ‘-Wp,-D_FORTIFY_SOURCE=3’
+          #
+          # even though CRAN itself sets that:
+          # https://www.stats.ox.ac.uk/pub/bdr/Rconfig/r-devel-linux-x86_64-fedora-clang
+          #
+          declare -i allowed_notes=1
+          bash .ci/run-r-cmd-check.sh \
+            "$(echo lightgbm_*.tar.gz)" \
```

Comment:
Maybe use this for the consistency?
```
"lightgbm_$(head -1 VERSION.txt).tar.gz"
```

Reply:
yeah I think you're right.

Fixed in https://github.com/microsoft/LightGBM/pull/6523/commits/f8cfe74f00764fa383082f813b80715cd2d2f14a.